Desktop/components/JASP/Widgets/EnginesWindow.qml

```diff
@@ -0,0 +1,106 @@
+import QtQuick			2.12
+import QtQuick.Window	2.12
+import QtWebEngine		1.8
+import JASP.Widgets		1.0
+import JASP.Controls	1.0
+
+Window
+{
+	id:					enginesWindow
+	width:				400 * jaspTheme.uiScale
+	height:				Screen.height
+	minimumWidth:		200 * preferencesModel.uiScale
+	minimumHeight:		minimumWidth
+	title:				qsTr("Engines Overview")
+	color:				jaspTheme.uiBackground
+	visible:			true
+	x:					0
+	y:					0
+
+	Shortcut { onActivated: enginesWindow.close();				sequences: ["Ctrl+W", Qt.Key_Close];							}
+
+
+	Connections
+	{
+		target:			mainWindow
+		function onCloseWindows() { enginesWindow.close(); }
+	}
+
+
+	ListView
+	{
+		anchors.fill:	parent
+		model:			engineSync
+
+
+		delegate:	Item
+		{
+
+			height:			150 * jaspTheme.uiScale
+			width:			ListView.view.width
+
+			Rectangle
+			{
+				color:					jaspTheme.uiBackground
+				border.color:			jaspTheme.uiBorder
+				border.width:			1
+				radius:					jaspTheme.borderRadius * 2
+
+				anchors.fill:			parent
+				anchors.margins:		jaspTheme.generalAnchorMargin
+
+				Text
+				{
+					text:				model.display + "\nState: " + model.engineState + "\nModule: " + model.module + (model.analysisStatus === "NA" ? "" : "\nAnalysis Status: " + model.analysisStatus)
+					font.pixelSize:		20 * jaspTheme.uiScale
+					font.bold:			true
+					anchors
+					{
+						left:			parent.left
+						leftMargin:		jaspTheme.generalAnchorMargin
+						verticalCenter: parent.verticalCenter
+					}
+				}
+
+				Rectangle
+				{
+					height: parent.height * 0.5
+					width:	height
+					radius: height
+
+					border.color:		jaspTheme.black
+					border.width:		6 * jaspTheme.uiScale
+					color:				model.idle ? "green" : model.analysisStatus !== "NA" ? jaspTheme.jaspBlue : model.idleSoon ? "yellow" : "orange"
+
+					anchors
+					{
+						right:			parent.right
+						rightMargin:	jaspTheme.generalAnchorMargin
+						verticalCenter:	parent.verticalCenter
+					}
+			}
+		}
+
+		JASPMouseAreaToolTipped
+		{
+			acceptedButtons:	Qt.MiddleButton
+
+			toolTipText:		qsTr("Middle mouse button to kill an engine.")
```

Comment:
Yes that is a very good point! I can add a little `x`-mark in the toprightcorner for instance? Or the red-circle-delete icon from the analyses-view?

The question I won't do because really there is nothing that could go wrong except for waiting slighlty longer for your analysis or whatever to finish. So that would just be useless clicking then.

Reply:
I think the red circle from the analysis should work, and it's also familiar. 

Are you saying that we shouldn't have the terminate functionality at all?

Reply:
No Im saying that the terminate functionality is very safe and no work will be lost no matter what.
So that leads me to conclude that having a warning (aka "upon click ask the user to make sure.") is unnecessary.